api/src/main/java/bio/terra/tanagra/service/underlay/Underlay.java

```diff
@@ -67,6 +69,15 @@
     return Optional.of(matching.get(0));
   }
 
+  /** Returns all the relationships that the entity is a member of. */
+  public Set<Relationship> getRelationshipsOf(Entity entity) {
+    return relationships().values().stream()
+        .filter(
+            relationship ->
+                relationship.entity1().equals(entity) || relationship.entity2().equals(entity))
```

Comment:
nit: seems this can be replaced by `relationship.hasEntity(entity)`

Reply:
done